docs/reference/commandline/commit.md

```diff
@@ -31,7 +31,7 @@ volumes mounted inside the container.
 By default, the container being committed and its processes will be paused
 while the image is committed. This reduces the likelihood of encountering data
 corruption during the process of creating the commit.  If this behavior is
-undesired, set the 'p' option to false.
+undesired, set the `-p` option to false.
```

Comment:
I feel this is clear without the change. I need to check on our standards and how it looks in other places, but I prefer something like the ``` the `pause` option``` or ```the `p` flag```.